extensions/measurement-tracking/src/_shared/createReportAsync.js

```diff
@@ -18,7 +18,8 @@ async function createReportAsync(servicesManager, dataSource, measurements) {
   try {
     const naturalizedReport = await DICOMSR.storeMeasurements(
       measurements,
-      dataSource
+      dataSource,
+      ['ArrowAnnotate']
```

Comment:
What meow?

Reply:
We need to encode additional findings as findings, and measurements as findingSites in the DICOM, there isn't a good way of doing this extensibly without passing in the list of what should be an annotation, happy for any suggestions.